src/main/java/com/kodilla/ecommercee/CartController.java

```diff
@@ -1,4 +1,47 @@
 package com.kodilla.ecommercee;
 
+
+import com.kodilla.ecommercee.domain.CartDto;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("/v1/cart")
 public class CartController {
+
+    @PostMapping(value = "createNewCart")
+    public GenericEntity createNewCart() {
+        return new GenericEntity("newCartDto");
+    }
+
+    @GetMapping(value = "getProducts")
+    public List<GenericEntity> getProductsFromCart (@PathVariable Long cartId) {
```

Comment:
Tutaj jednak trzeba dać to jako zwyczajny argument. Ogólnie zauważylem że w poprzednich modułach nie jest używany `@PathVariable` do tego więc żeby nie wprowadzać zamieszania, nie będziemy tego używać

Reply:
Dzięki. Rozumiem, że w ogóle wyrzucamy `@PathVariable`?